corehq/pillows/domain.py

```diff
@@ -53,7 +53,7 @@ def change_transform(self, doc_dict):
         sub =  Subscription.objects.filter(
                 subscriber__domain=doc_dict['name'],
                 is_active=True)
-        doc_dict['deployment'] = doc_dict.get('deployment', None) or {}
+        doc_ret['deployment'] = doc_dict.get('deployment', None) or {}
```

Comment:
Going with your suggestion @millerdev. Better to avoid the original side effects, and I'm sure this is what the author (@emord ?) intended.